src/JildertMiedema/LaravelPlupload/Receiver.php

```diff
@@ -91,7 +92,7 @@ public function hasChunks()
 
     public function receive($name, Closure $handler)
     {
-        $response = array();
+        $response = [];
```

Comment:
You are right, not needed at all. I did not rember to change it... probably when I'm importing the namespaces with the build in function in PhpStorm changes this line like that. Should I fix it and submit a new pull request?